NuGet.config

```diff
@@ -7,6 +7,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json"/>
```

Comment:
This must NOT be merged. I'll mirror the packages following https://github.com/dotnet/arcade/blob/main/Documentation/MirroringPackages.md once the PR is closer.